packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -138,7 +138,7 @@ export class ProjectModel {
             .select('project_uuid', 'name', 'project_type')
             .where('organization_id', orgs[0].organization_id);
         if (projects.length === 0) {
-            throw new NotExistsError('No project exists');
+            return [];
```

Comment:
@IrakliJani @rephus Looks like you have opposite changes in your open PR's

https://github.com/lightdash/lightdash/pull/3337/files#r983189676